developers/integrations/llm-frameworks/composio/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Composio
+sidebar_position: 1
+image: og/integrations/home.jpg
+---
+
+[Composio](https://docs.composio.dev/introduction/intro/overview) uses function calling for managing and integrating tools with language models and AI agents.  
+
+## Composio and Weaviate 
+With Weaviate's retrieval, you can make the agent more personalized and context-aware. 
+
+The integration is supported through our LangChain vector store. You need to have a running Weaviate instance and create the vector store with: 
```

Comment:
Consider

The integration is supported through our LangChain vector store. 

To configure the integration, create the vector store in your Weaviate instance: